panel/src/components/Forms/Calendar.vue

```diff
@@ -40,7 +40,7 @@
       <tfoot>
         <tr>
           <td class="k-calendar-today" colspan="7">
-            <k-button @click="go('today')">{{ $t("today") }}</k-button>
+            <k-button @click="go('today')">{{ $t("field.date.month.current") }}</k-button>
```

Comment:
I think we should rename the special `today` date in the `go()` method to `current-month` as well to be consistent.